marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/flow/job-query-lib.sjs

```diff
@@ -24,16 +24,20 @@ function findStepResponses(query) {
   const sortColumn = query.sortColumn;
   const sortDirection = query.sortDirection;
   const response = {};
+  const orderByConstraint = [];
 
-  let orderByConstraint = op.desc('startTime');
   if(sortColumn) {
-    orderByConstraint = sortDirection === 'ascending' ? op.asc(sortColumn) : op.desc(sortColumn);
+    orderByConstraint.push(sortDirection === 'ascending' ? op.asc(sortColumn) : op.desc(sortColumn));
+  }
+
+  if(sortColumn !== 'startTime') {
+    orderByConstraint.push(op.desc('startTime'));
   }
 
   const totalCountQuery = 'select count(*) as total from Job.StepResponse';
   const jobsDataQuery = 'select Job.StepResponse.stepName as stepName,' +
       'Job.StepResponse.stepDefinitionType as stepDefinitionType,' +
-      'Job.StepResponse.stepStatus as status,' +
+      'Job.StepResponse.jobStatus as status,' +
```

Comment:
I don't think "status" is good to use here, because that begs the question - Is this the job status, or is it the step status? 

Given that this is a StepResponse, I think it's expected that "status" would refer to the step status. But it's not - it's the job status. So this should really be "as jobStatus".

Reply:
Also, the fact that this code changed, but there's no accompanying change in a test, means that we don't have any test coverage on this column. That should be easy to add to your test below - you'd just verify that the expected job status is returned.